include/ccl_emu17.h

```diff
@@ -11,11 +11,12 @@ CCL_BEGIN_DECLS
  * Emulator power spectrum
  * Obtain P(k,z) [Mpc^3] for a given set of input parameters.
  * @param xstarin vector of input parameters for the emulator, including redshift. 
+ * @param nkemu number of elements to which Pkemu has been allocated.
```

Comment:
What does this mean?

Reply:
It's the size of the next argument (`Pkemu`)

Reply:
Ahhh. The doc string confused me.